action/create_vm.go

```diff
@@ -47,20 +52,33 @@ func (a CreateVM) Run(agentID string, stemcellCID StemcellCID, cloudProps bslcvm
 	return VMCID(vm.ID()).String(), nil
 }
 
-func (a CreateVM) updateCloudProperties(cloudProps bslcvm.VMCloudProperties) {
-	if cloudProps.StartCpus > 1 {
-		a.vmCloudProperties.StartCpus = cloudProps.StartCpus
+func (a CreateVM) UpdateCloudProperties(cloudProps *bslcvm.VMCloudProperties) {
```

Comment:
where is the new unit test or change in existing unit test for this set of changes?